src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Media/GradientBrushTests/GradientsPage.xaml

```diff
@@ -0,0 +1,248 @@
+							<GradientStop Color="LightBlue" Offset="0.6" />
+							<GradientStop Color="Blue" Offset="0.8" />
+							<GradientStop Color="LightGray" Offset="1" />
+						</not_win:RadialGradientBrush>
+					</Ellipse.Fill>
+				</Ellipse>
+			</StackPanel>
+			<TextBlock FontSize="15">Radial Gradient as border/stroke on a Rectangle/ Border / Ellipse </TextBlock>
+			<StackPanel Orientation="Horizontal" Spacing="10">
+				<Rectangle Width="100" Height="100" StrokeThickness="40" Fill="DarkGray">
+					<Rectangle.Stroke>
+						<not_win:RadialGradientBrush FallbackColor="Red">
+							<GradientStop Color="Blue" Offset="0.0" />
+							<GradientStop Color="Yellow" Offset="0.2" />
+							<GradientStop Color="LimeGreen" Offset="0.4" />
+							<GradientStop Color="LightBlue" Offset="0.6" />
+							<GradientStop Color="Blue" Offset="0.8" />
+							<GradientStop Color="LightGray" Offset="1" />
+						</not_win:RadialGradientBrush>
+					</Rectangle.Stroke>
+				</Rectangle>
+
+				<Border Width="100" Height="100" BorderThickness="40" Background="DarkGray">
+					<Border.BorderBrush>
+						<not_win:RadialGradientBrush FallbackColor="Red">
+							<GradientStop Color="Blue" Offset="0.0" />
+							<GradientStop Color="Yellow" Offset="0.2" />
+							<GradientStop Color="LimeGreen" Offset="0.4" />
+							<GradientStop Color="LightBlue" Offset="0.6" />
+							<GradientStop Color="Blue" Offset="0.8" />
+							<GradientStop Color="LightGray" Offset="1" />
+						</not_win:RadialGradientBrush>
+					</Border.BorderBrush>
+				</Border>
+				<Ellipse Width="100" Height="100" StrokeThickness="40" Fill="DarkGray">
+					<Ellipse.Stroke>
+						<not_win:RadialGradientBrush FallbackColor="Red">
+							<GradientStop Color="Blue" Offset="0.0" />
+							<GradientStop Color="Yellow" Offset="0.2" />
+							<GradientStop Color="LimeGreen" Offset="0.4" />
+							<GradientStop Color="LightBlue" Offset="0.6" />
+							<GradientStop Color="Blue" Offset="0.8" />
+							<GradientStop Color="LightGray" Offset="1" />
+						</not_win:RadialGradientBrush>
+					</Ellipse.Stroke>
+				</Ellipse>
+				<Ellipse Width="150" Height="100" StrokeThickness="40" Fill="DarkGray">
+					<Ellipse.Stroke>
+						<not_win:RadialGradientBrush FallbackColor="Red">
+							<GradientStop Color="Blue" Offset="0.0" />
+							<GradientStop Color="Yellow" Offset="0.2" />
+							<GradientStop Color="LimeGreen" Offset="0.4" />
+							<GradientStop Color="LightBlue" Offset="0.6" />
+							<GradientStop Color="Blue" Offset="0.8" />
+							<GradientStop Color="LightGray" Offset="1" />
+						</not_win:RadialGradientBrush>
+					</Ellipse.Stroke>
+				</Ellipse>
+			</StackPanel>
+			<TextBlock FontSize="15">Linear Gradient as background on a Rectangle + Border (relative + absolute)</TextBlock>
+			<StackPanel Orientation="Horizontal" Spacing="10">
+				<Rectangle Width="100" Height="100">
+					<Rectangle.Fill>
+						<LinearGradientBrush not_win:FallbackColor="Red">
```

Comment:
I would have the `LinearGradientBrush` and `RadialGradientBrush` samples in separate pages.